backend/infrahub/core/node/__init__.py

```diff
@@ -550,6 +551,8 @@ async def to_graphql(
 
         for field_name in field_names:
             if field_name == "__typename":
+                # Note we already store kind within KIND_GRAPHQL_FIELD_NAME, understanding if/why filling
```

Comment:
this is a field that the user can query so we need to support that
the frontend is using it heavily